### DISCUSSION TITLE
Determining Fan Locations

### DISCUSSION THREAD

**Discussion by drdrumlord** [EvanMulawski/FanControl.CorsairLink#140] [General]
iCue software was causing stuttering in my games, so I switched to Fan Control with this plugin, and everything is working well now. However, I'm not sure how to identify each individual fan in the software. I have 10 case fans, and I would like to be able to identify their placement so I can have better push-pull control. Any advice on how to identify individual fans when using Fan Control?

**Comment by EvanMulawski** [comment]
Hi @drdrumlord, once they are set up in Fan Control you can go through them and turn Manual Control on and set the value to 0%/100% to identify the fan and update the name accordingly.

![image](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/1044791/46964b81-d0fc-4413-aed8-1d4e61c3a0fc)

**Reply by drdrumlord** [reply]
Got it, thank you! I'm new to this world of PC building, so it freaked me out to turn all of my fans down and test each one, but your comment was very helpful and it worked. Now I have all the fans assigned correctly. Thanks again!!
